man/bst.1.scd

```diff
@@ -6,7 +6,7 @@ bst - run executables in their own spacetime.
 
 # SYNOPSIS
 
-bst [options] [--] [executable [args...]]
+bst [options] [--] [VAR=value...] [executable [args...]]
```

Comment:
This usage line suggests that `bst FOO=bar` would be legal.  Is it?  I guess it just unshares and starts a shell with that variable set?

Reply:
Yeah -- `bst` defaults the executable to $SHELL, so it would indeed start a shell with that variable set.